subsys/bluetooth/mesh/adv_legacy.c

```diff
@@ -97,6 +97,9 @@ static inline void adv_send(struct net_buf *buf)
 
 	uint64_t time = k_uptime_get();
 
+	/* Avoiding build warning when debug not enabled. */
+	(void) time;
```

Comment:
Better to guard the variable instead in my opinion.

Reply:
？

Reply:
You can guard the variable with `CONFIG_BT_MESH_DEBUG_ADV` so that it only used in that configuration

```
#if defined(CONFIG_BT_MESH_DEBUG_ADV)
	uint64_t time = k_uptime_get();
#endif /* CONFIG_BT_MESH_DEBUG_ADV */
```

This will not only fix the warning, but also make it clear that this variable is only used for a debug configuration.